external-plugins/coverage/plugin/handler/handler_test.go

```
@@ -506,3 +531,187 @@ var _ = Describe("Handle", func() {
 })
+
+var _ = Describe("LoadConfig", func() {
+	var writeConfig func(content string) string
+
+	BeforeEach(func() {
+		writeConfig = func(content string) string {
+			f, err := os.CreateTemp("", "coverage-config-*.yaml")
+			Expect(err).NotTo(HaveOccurred())
+			_, err = f.WriteString(content)
+			Expect(err).NotTo(HaveOccurred())
+			f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)